### DISCUSSION TITLE
Need clarification on how it works on a production stack

### DISCUSSION THREAD

**Discussion by Rollbacke** [sablierapp/sablier#382] [Q&A] [answered]
Hey everyone,

I've a production docker stack with ~70 containers.
I'm happy to see the #346 pull which provide the `provider.auto-stop-on-startup` option.

Now, I can confirm, my sablier supervised containers are shut down by sablier after its start.
But, if I do a `docker compose up -d` on my docker stack, and there is no update on sablier. The sablier container is not restarted, my containers are all started (especially the ones that were shut down by sablier).
Does sablier shut down the containers again? Or will it wait for the configured time?
Or do I've to make a first request like before for sablier to decide to start the timer?

Thanks !

**Comment by acouvreur** [comment, marked as answer]
Hello @Rollbacke 

The auto start currently only shut down the instance on start, once.

You can open an issue so that Sablier can track startup event not triggered by Sablier so it can shutdown containers started by an external actor automatically.

**Comment by Rollbacke** [comment]
That's what I understood.
I'll do that as soon as I have a moment ;)

Thanks.
